silverback/cluster/types.py

```diff
@@ -1,9 +1,12 @@
+from __future__ import annotations
```

Comment:
Idk if I'm blind or if this isn't used anywhere

Reply:
You're not blind, but this is... different. The act of importing this allows older python versions to "support" modern type annotations. In this case, it's the union operator.

Reply:
sweet